test/integration-tests/configs/single-producer/serverless.yml

```diff
@@ -0,0 +1,24 @@
+service: ${env:PLUGIN_IDENTIFIER}-single-producer-${env:RANDOM_STRING}
+
+provider:
+  lambdaHashingVersion: 20201221
+  name: aws
+  runtime: python3.8
+  region: us-west-2
+  stage: test
+  role: arn:aws:iam::646102706174:role/envoy_admin_ui_role
```

Comment:
This should be removed. I believe `role` is an optional field. Also we do not want to commit AWS account IDs or AWS account role names in this repo since this is an open source repo